source/extensions/filters/http/query_parameter_mutation/config.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <string>
+
+#include "envoy/extensions/filters/http/query_parameter_mutation/v3/config.pb.h"
+#include "envoy/extensions/filters/http/query_parameter_mutation/v3/config.pb.validate.h"
+
+#include "source/extensions/filters/http/common/factory_base.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace QueryParameterMutation {
+
+class Factory : public Extensions::HttpFilters::Common::FactoryBase<
+                    envoy::extensions::filters::http::query_parameter_mutation::v3::Config> {
```

Comment:
Seems you can use an alias name to represent  the `envoy::extensions::filters::http::query_parameter_mutation::v3::Config`.

Like:
```
using ProtoQueryParameterMutation = envoy::extensions::filters::http::query_parameter_mutation::v3::Config;
```